blocks/form/form.css

```diff
@@ -0,0 +1,68 @@
+.form {
+  --input-border: #d6d6d6;
+}
+
+.form-wrapper {
+  width: 100%;
+  margin: 0 auto;
+  max-width: var(--body-max-width);
+  padding: 0 var(--body-padding);
```

Comment:
Would it be possible to have this padding for everything, even the heading so that everything has the same distance from the border?
<img width="660" alt="Screenshot 2023-08-23 at 17 16 48" src="https://github.com/hlxsites/bitdefender/assets/30388999/18fc3ebd-d5d0-4f19-b4a4-3890ae219a1a">